docs/EARLY_TESTERS.md

```diff
@@ -0,0 +1,33 @@
+# EARLY TESTERS PROGRAMME
+
+## What is it?
+
+The early testers programme allows members of the community to self-volunteer to help test `go-ipfs` release candidates. While we invite the _entire_ community to help test releases, members of the early testers program are expected to participate directly and actively in every release.
+
+## What are the expectations?
+
+Members of the early tester program are expected to work closely with us to:
+
+* Provide high quality, actionable feedback.
+* Work directly with us to debug regressions in the release.
+* Help ensure a rock-solid, timely release.
+
+We will ask early testers to participate at two points in the process:
+
+* When go-ipfs enters the second release stage (public beta), early testers will be asked to test go-ipfs on non-production infrastructure. This may involve things like:
+  - Running integration tests against the release candidate.
+  - Running simulations/benchmarks on the release candidate.
+  - Manually testing the release candidate to check for regressions.
+* When go-ipfs enters the third release stage (soft release), early testers will be asked to partially deploy the release candidate to production infrastructure. Release candidates at this stage are expected to be identical to the final release. However, this stage allows the go-ipfs team to fix any last-minute regressions without cutting an entirely new release.
+
+## Who has signed up?
+
+- [ ] Infura (@MichaelMure)
+- [ ] Textile (@sanderpick)
+- [ ] Pinata (@obo20)
+- [ ] RTrade (@postables)
+- [ ] QRI (@b5)
```

Comment:
@Stebalien did you get a confirmation from all of these after getting this PR up? It would be good to double check given that we only have now a description of what it is.

Reply:
Will do.